### PR TITLE
JARS-16 Adding collections panel to the resource list on the left side

### PR DESCRIPTION
**Pull request by sanjanavsdvn**

### CODE REVIEW

**Comment by erickpeirson** [comment]
Great start! Right off the bat I see we need some work on the layout (will look closer momentarily)

image

File: cookies/templates/resources.html [hunk 1]
```diff
@@ -52,21 +52,37 @@ <h4 class="panel-title">
     </div>
 </div>
 
-<div class="collection_resources">
-    <div class="pagination container text-center">
-        {% paginate %}
+<div class = "row">
+    <div class = "col-sm-1">
```
Comment (erickpeirson) [inline cookies/templates/resources.html]: Good idea to use the ``sm`` breakpoint. But you can use more space than that -- try ``col-sm-3`` or ``col-sm-4``

File: cookies/templates/resources.html [hunk 2]
```diff
@@ -52,21 +52,37 @@ <h4 class="panel-title">
     </div>
 </div>
 
-<div class="collection_resources">
-    <div class="pagination container text-center">
-        {% paginate %}
+<div class = "row">
```
Comment (erickpeirson) [inline cookies/templates/resources.html]: This is just a stylistic point: in the templates, don't put whitespace around the ``=`` sign in HTML attributes.

File: cookies/templates/resources.html [hunk 3]
```diff
@@ -52,21 +52,37 @@ <h4 class="panel-title">
...
+    <div class = "col-sm-1">
+        <div class = "panel panel-info">
+            <h5 class = "panel-heading">Collections</h5>
```
Comment (erickpeirson) [inline cookies/templates/resources.html]: Using an ``h5`` tag creates a lot of ugly padding. Try this instead:

```html
<div class="panel panel-info">
    <div class="panel-heading">
        <span class="h5">Collections</span>
    </div>
    ...
</div>
```

**Comment by erickpeirson** [comment]
@sanjanavsdvn Take a look at my comments in the "files changed" tab. Well done, just a few small tweaks.

**Comment by sanjanavsdvn** [comment]
Looked at your comments and made the changes. Please take a look.

**Comment by erickpeirson** [comment]
@sanjanavsdvn Looks great! Will merge once Travis is done.

**Pull request merged**